catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfPreStoragePlugin.java

```diff
@@ -210,14 +211,23 @@ private ContentItem createDerivedImage(String id, String qualifier, BufferedImag
         return null;
     }
 
-    protected String buildDerivedImageTitle(String title, String qualifier) {
+    String buildDerivedImageTitle(String title, String qualifier) {
+        String derivedImageTitle;
         String rootFileName = FilenameUtils.getBaseName(title);
 
-        return (StringUtils.isNotBlank(rootFileName) &&
-                StringUtils.isNotBlank(rootFileName.replaceAll("[^A-Za-z0-9]", ""))) // any human readable content
-                ? String.format(DERIVED_IMAGE_FILENAME_PATTERN, qualifier,
-                        rootFileName.replaceAll(INVALID_FILENAME_CHARACTER_REGEX, ""), JPG)
-                : String.format("%s.%s", qualifier, JPG);
+        // title must contain some alphanumeric, human readable characters, or use default filename
+        if (StringUtils.isNotBlank(rootFileName)
+                && StringUtils.isNotBlank(rootFileName.replaceAll("[^A-Za-z0-9]", ""))) {
```

Comment:
Should have asked this before...the only difference between this regex and the other one you're using is the underscore. Is there a reason for that? Could you just use the const here?